src/App.css

```
@@ -25,263 +25,462 @@ button {
-  box-shadow: 0 9px 25px rgba(33, 79, 54, 0.24);
+  background: #245e3a;
+  box-shadow: 0 7px 18px rgba(36, 94, 58, 0.21);
 }
 
-.brand-mark span {
-  width: 6px;
-  border-radius: 4px;
+.brand-mark > span {
+  width: 4px;
+  border-radius: 3px;
   background: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)